lib/tasks/example_data.rake

```diff
@@ -90,6 +93,7 @@ def staff_members
       change_work_history_permission: false,
       reverse_decision_permission: false,
       support_console_permission: false,
+      verification_permission: false,
```

Comment:
Could we add a new "admin" persona who does have permission to perform verification, but nothing else?